spec/unit/xlsx/xform/book/defined-name-xform.spec.js

```diff
@@ -8,17 +8,17 @@ var expectations = [
   {
     title: 'Defined Names',
     create: function() { return new DefinedNameXform(); },
-    preparedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1']},
+    preparedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1'], formula: ''},
     xml: '<definedName name="foo">bar!$A$1:$C$1</definedName>',
-    parsedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1']},
+    parsedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1'], formula: ''},
     tests: ['render', 'renderIn', 'parse']
   },
   {
     title: 'Print Area',
     create: function() { return new DefinedNameXform(); },
-    preparedModel: {name: '_xlnm.Print_Area', localSheetId: 0, ranges: ['bar!$A$1:$C$10']},
+    preparedModel: {name: '_xlnm.Print_Area', localSheetId: 0, ranges: ['bar!$A$1:$C$10'], formula: ''},
```

Comment:
input data in tests should not be modified, it causes lost possibility to test backward compatibility